addons/mail/models/mail_channel.py

```diff
@@ -1153,6 +1158,28 @@ def channel_create(self, name, privacy='public'):
         self.env['bus.bus'].sendone((self._cr.dbname, 'res.partner', self.env.user.partner_id.id), channel_info)
         return channel_info
 
+    @api.model
+    def group_chat_create(self, partners_to):
+        """ Create a private group chat with at least 3 partners.
+            :param partners_to : list of res.partner ids to add to the conversation
+            :returns: channel_info of the created or existing channel
+            :rtype: dict
+        """
+        if self.env.user.partner_id.id not in partners_to:
+            partners_to.append(self.env.user.partner_id.id)
```

Comment:
I think this was a mistake from the original method that we don't want to copy. Prefer always giving the current user as param instead of forcing it.